sdk/search/Azure.Search.Documents/tests/Azure.Search.Documents.Tests.csproj

```diff
@@ -31,13 +31,20 @@
   <ItemGroup>
     <PackageReference Include="Azure.Storage.Blobs" />
     <PackageReference Include="Azure.Identity" />
-
-    <!-- TODO: Remove this when https://github.com/Azure/azure-sdk-for-net/issues/10592 is resolved. -->
-    <PackageReference Condition="'$(UseAzureCoreExperimentalSpatial)' != 'true'" Include="Microsoft.Spatial" />
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Update="Samples\HelloWorld.Data.cs" DependentUpon="Samples\HelloWorld.cs" />
+    <Compile Update="Samples\Sample01_HelloWorld.Data.cs" DependentUpon="Sample01_HelloWorld.cs" />
+  </ItemGroup>
+
+  <!-- Conditionally compile FieldBuilder and related source -->
+  <ItemGroup Condition="'$(UseFieldBuilder)' == 'true'">
+    <Compile Remove="Samples\FieldBuilder\**" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(UseFieldBuilder)' != 'true'">
+    <Compile Remove="SearchableFieldAttributeTests.cs" />
+    <Compile Remove="SimpleFieldAttributeTests.cs" />
+    <Compile Remove="Samples\Sample2_FieldBuilder.cs" />
   </ItemGroup>
```

Comment:
When it's true we want to remove them?

Reply:
Given all .cs is included by default, it's far easier to remove what you don't want.

Reply:
Oh, sorry, this is removing the "FieldBuilder as a Sample" code, not FieldBuilder itself.  I was thinking that would get deleted but it's probably best to leave like you have for now.